wpeverest/user-registration
# Feature - Edit profile form shortcode

### All Submissions:






### Changes proposed in this Pull Request:


Users are asking for a separate edit-profile page shortcode. We can provide a [user_registration_edit_profile] shortcode that renders the edit-profile page of my account anywhere the user wishes.

### How to test the changes in this Pull Request:

1. Navigate to pages and add a new page.
2. Add the title and click on add block ie (+) sign and click on the shortcode.
3. Paste the shortcode [user_registration_edit_profile] and publish it and you can see the edit profile page of my account on the frontend.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Edit profile form shortcode.
